static/src/stylesheets/module/facia-garnett/_item.scss

```
@@ -540,24 +540,37 @@ $block-height: 58px;
 }
 
 .fc-item__slideshow {
-    picture {
+    figure {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        margin: 0;
+
         @include mq($until: tablet) {
-            &:nth-child(1n+2) img {
+            &:nth-child(1n+2) {
                 display: none;
             }
         }
-    }
 
-    img {
         @include mq(tablet) {
             animation-timing-function: linear;
             animation-iteration-count: infinite;
             animation-direction: normal;
             opacity: 0;
         }
     }
-}
 
+    figcaption {
+        @include fs-bodyHeading(1);
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%,  rgba(0, 0, 0, .8) 100%);
```

Suggestion: `to bottom`

```suggestion
        background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 0%,  rgba(0, 0, 0, .8) 100%);
```